packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md

```diff
@@ -109,11 +109,88 @@ You can build and run UT with:
 yarn test
 ```
 
-## Vendor extensions
+## Manage dates
+
+### The timezone issue
+Managing dates with timezones has always been a bit painful in front end applications.
+Let's give a concrete example to understand the problem:
+In the case of an API returning the date and hour of your flight : 2023-07-10T00:37:00.000+07:00
+The timezone sent is the one from the airport, here GMT+7. If you just use the Date(), the computer browser will convert this in its own timezone.
+You will end up having for example if the user is in GMT+2 : 2023-07-09T19:37:00.000+02:00.
+You don't want that, you want the exact date time of the flight, at the airport timezone, not the user one.
+However, sometimes you might need this timezone information, for example to be able to display to the user that the flight is in X hours (need both airport and user timezone).
+
+### Solution proposed to remove the timezone: utils.DateTime
+That's why the default Date behavior is to get rid of the timezone, by replacing Date by utils.Date.
```

Comment:
```suggestion
As we need to get rid the timezones more often than not, this will be our default behaviour. This is why, we introduced the utils.Date object to replace Date in those use cases.
```

Reply:
Date or DateTime ?